src/main/java/reposense/RepoSense.java

```diff
@@ -76,6 +85,22 @@ public static void main(String[] args) {
         }
     }
 
+    /**
+     * Returns a list of relevant repo folders to be zipped up later
+     * @param sourcePath
+     * @param configs
+     * @return
+     */
+    private static HashSet<Path> getRelativeFolders(Path sourcePath, List<RepoConfiguration> configs) {
```

Comment:
I am not so sure it is a good place to put the method here.  I will leave this comment here to see whether the seniors have other suggestions.

Reply:
My rough idea is to write a new class called `ZipGenerator` to handle below sentences:
```
    HashSet<Path> relevantPaths = getRelativeFolders(cliArguments.getOutputFilePath(), configs);
    relevantPaths.add(Paths.get(cliArguments.getOutputFilePath().toAbsolutePath().toString(),
                SummaryReportJson.SUMMARY_JSON_FILE_NAME));
     FileUtil.zipRelativeFiles(relevantPaths, cliArguments.getOutputFilePath().toAbsolutePath(), ".json");
```
But let us to wait for the seniors to decide.

Reply:
Alright noted